macros/src/test/scala/gestalt/macros/DefMacroTest.scala

```diff
@@ -211,4 +211,9 @@ class DefMacroTest extends TestSuite {
     assert(trees.some3Explicit() == Some(3))
     assert(trees.some3ExplicitTyped() == Some(3))
   }
+
+  test("white box") {
+    val x = Whitebox.gimmelist(2)
```

Comment:
Just tried locally, explicit annotation doesn't work, which is surprising to programmers. While it's possible to remove the surprise, it's only worth the effort if whitebox should be supported.